tests/integration/devfile/cmd_devfile_deploy_test.go

```diff
@@ -64,13 +66,23 @@ var _ = Describe("odo devfile deploy command tests", func() {
 					Expect(out).To(ContainSubstring("quay.io/unknown-account/myimage"))
 				})
 			})
+			It("should successfully record telemetery information correctly", func() {
+				td := helper.GetTelemetryDebugData()
+				Expect(td.Event).To(ContainSubstring("odo deploy"))
+				Expect(td.Properties.Success).To(BeTrue())
+				Expect(td.Properties.Error == "").To(BeTrue())
+				Expect(td.Properties.ErrorType == "").To(BeTrue())
+				Expect(td.Properties.CmdProperties[segment.ComponentType]).To(ContainSubstring("nodejs"))
+				Expect(td.Properties.CmdProperties[segment.Language]).To(ContainSubstring("javascript"))
+				Expect(td.Properties.CmdProperties[segment.ProjectType]).To(ContainSubstring("nodejs"))
+			})
 
 			When("deleting previous deployment and switching kubeconfig to another namespace", func() {
 				var otherNS string
 				BeforeEach(func() {
 					helper.Cmd("odo", "delete", "component", "--name", cmpName, "-f").ShouldPass()
 					output := commonVar.CliRunner.Run("get", "deployment", "-n", commonVar.Project).Err.Contents()
-					Expect(string(output)).To(ContainSubstring("No resources found in " + commonVar.Project + " namespace."))
+					Expect(string(output)).To(ContainSubstring("No resources found"))
```

Comment:
Why have you removed this?

Reply:
It is failing the test

Reply:
Can you share the output of the test failure?

Reply:
```bash
make test-cmd-devfile-deploy 
go install -mod=vendor -ldflags="-X github.com/redhat-developer/odo/pkg/version.GITCOMMIT=5f3270420" ./cmd/odo/
GOFLAGS='-mod=vendor' go run -mod=vendor github.com/onsi/ginkgo/ginkgo  -randomizeAllSpecs -slowSpecThreshold=120 -timeout 14400s -nodes=4 -focus="odo devfile deploy command tests" tests/integration/devfile/
Running Suite: Devfile Suite
============================
Random Seed: 1648112830 - Will randomize all specs
Will run 169 specs

Running in parallel across 4 nodes

SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•S•SSSSSSSSSSSSSSSSSS•S
------------------------------
• Failure in Spec Setup (BeforeEach) [1.161 seconds]
odo devfile deploy command tests
/home/mzee1000/workbench/work/odo/odo/tests/integration/devfile/cmd_devfile_deploy_test.go:13
  when using a devfile.yaml containing a deploy command
  /home/mzee1000/workbench/work/odo/odo/tests/integration/devfile/cmd_devfile_deploy_test.go:41
    when running odo deploy
    /home/mzee1000/workbench/work/odo/odo/tests/integration/devfile/cmd_devfile_deploy_test.go:50
      when deleting previous deployment and switching kubeconfig to another namespace
      /home/mzee1000/workbench/work/odo/odo/tests/integration/devfile/cmd_devfile_deploy_test.go:80
        should run odo deploy on initial namespace [BeforeEach]
        /home/mzee1000/workbench/work/odo/odo/tests/integration/devfile/cmd_devfile_deploy_test.go:94

        Expected
            <string>: No resources found.
            
        to contain substring
            <string>: No resources found in cmd-devfile-deploy-test94bhg namespace.

        /home/mzee1000/workbench/work/odo/odo/tests/integration/devfile/cmd_devfile_deploy_test.go:85
------------------------------
•

Summarizing 1 Failure:

[Fail] odo devfile deploy command tests when using a devfile.yaml containing a deploy command when running odo deploy when deleting previous deployment and switching kubeconfig to another namespace [BeforeEach] should run odo deploy on initial namespace 
/home/mzee1000/workbench/work/odo/odo/tests/integration/devfile/cmd_devfile_deploy_test.go:85

Ran 5 of 169 Specs in 2.308 seconds
FAIL! -- 4 Passed | 1 Failed | 0 Pending | 164 Skipped


Ginkgo ran 1 suite in 5.227245596s
Test Suite Failed
exit status 1
make: *** [Makefile:200: test-cmd-devfile-deploy] Error 1
```

Reply:
@valaparthvi  ^

Reply:
This is weird. This is the output that I get.
```sh
➜  k get deploy               
No resources found in default namespace.
➜  k get deploy -n default             
No resources found in default namespace.
➜ k get deploy -n myproject
No resources found in myproject namespace.
```
And how come this was passing earlier, but not now. I'll test it on the main branch.

Reply:
They pass on my machine in both the branches.

Reply:
Yea it is strange behavior indeed/ I have no clue why it is happening that way either.
Anycase I am removing the change. I will figure out what is going on later